perfdash/parser.go

```diff
@@ -137,3 +137,58 @@ func parseRequestCountData(data []byte, buildNumber int, testResult *BuildData)
 		}
 	}
 }
+
+type metricValue struct {
```

Comment:
You probably shouldn't need to redefine a struct here. It should already be present somewhere in kubernetes/kubernetes code. Just try to import that package.

Reply:
I will add import, however I want to do this in a separate pull request. It requires changes in godep dependencies.

Reply:
sgtm

Reply:
Can you please add a TODO above the struct saying that we should eventually replace it with the dependency?

Reply:
done